go.mod

```diff
@@ -9,11 +9,11 @@ require (
 	github.com/stretchr/testify v1.8.2
 	go.uber.org/zap v1.24.0
 	google.golang.org/grpc v1.54.0
-	k8s.io/api v0.27.1
-	k8s.io/apimachinery v0.27.1
-	k8s.io/client-go v0.27.1
-	k8s.io/klog/v2 v2.100.1
-	k8s.io/kubelet v0.27.1
+	k8s.io/api v0.26.3
```

Comment:
any reason for using an earlier version?

Reply:
Hey @ejalberti thank you so much for reaching out regarding the device plugin.  I was wondering if you would be able to share more details regarding what the plugins are, what test case and environment that and any other general information you can share with us, Thanks a million.